core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -121,282 +121,282 @@ public function schema_changes_before_migration()
+					  CNT_cur_sign varchar(45) COLLATE utf8_bin DEFAULT '$',
+					  CNT_cur_sign_b4 tinyint(1) DEFAULT '1',
+					  CNT_cur_dec_plc tinyint(3) unsigned NOT NULL DEFAULT '2',
+					  CNT_cur_dec_mrk varchar(1) COLLATE utf8_bin NOT NULL DEFAULT '.',
+					  CNT_cur_thsnds varchar(1) COLLATE utf8_bin NOT NULL DEFAULT ',',
+					  CNT_tel_code varchar(12) COLLATE utf8_bin DEFAULT NULL,
+					  CNT_is_EU tinyint(1) DEFAULT '0',
+					  CNT_active tinyint(1) DEFAULT '0',
 					  PRIMARY KEY  (CNT_ISO)";
         $this->_table_is_new_in_this_version($table_name, $sql, 'ENGINE=InnoDB');
         $table_name = 'esp_datetime';
-        $sql = "DTT_ID INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
-				  EVT_ID BIGINT(20) UNSIGNED NOT NULL,
-				  DTT_EVT_start DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
-				  DTT_EVT_end DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
-				  DTT_reg_limit MEDIUMINT(8) DEFAULT -1,
-				  DTT_sold MEDIUMINT(8) UNSIGNED DEFAULT 0,
-				  DTT_is_primary TINYINT(1) UNSIGNED NOT NULL DEFAULT 0,
-				  DTT_order MEDIUMINT(3) UNSIGNED DEFAULT 0,
-				  DTT_parent INT(10) UNSIGNED DEFAULT 0,
-				  DTT_deleted TINYINT(1) UNSIGNED NOT NULL DEFAULT 0,
+        $sql = "DTT_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+				  EVT_ID bigint(20) unsigned NOT NULL,
+				  DTT_EVT_start datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+				  DTT_EVT_end datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+				  DTT_reg_limit mediumint(8) DEFAULT -1,
+				  DTT_sold mediumint(8) unsigned DEFAULT 0,
+				  DTT_is_primary tinyint(1) unsigned NOT NULL DEFAULT 0,
+				  DTT_order mediumint(3) unsigned DEFAULT 0,
+				  DTT_parent int(10) unsigned DEFAULT 0,
+				  DTT_deleted tinyint(1) unsigned NOT NULL DEFAULT 0,
 						PRIMARY KEY  (DTT_ID),
 						KEY EVT_ID (EVT_ID),
 						KEY DTT_is_primary (DTT_is_primary)";
         $this->_table_is_new_in_this_version($table_name, $sql, 'ENGINE=InnoDB');
         $table_name = 'esp_event_meta';
         $sql = "
-			EVTM_ID INT NOT NULL AUTO_INCREMENT,
-			EVT_ID BIGINT(20) UNSIGNED NOT NULL,
-			EVT_display_desc TINYINT(1) UNSIGNED NOT NULL DEFAULT 1,
-			EVT_display_ticket_selector TINYINT(1) UNSIGNED NOT NULL DEFAULT 1,
-			EVT_visible_on DATETIME NOT NULL DEFAULT '0000-00-00 00:00:00',
-			EVT_default_registration_status VARCHAR(3),
-			EVT_phone VARCHAR(45) DEFAULT NULL,
-			EVT_additional_limit TINYINT UNSIGNED NULL,
-			EVT_member_only TINYINT(1) UNSIGNED NOT NULL DEFAULT 0,
-			EVT_allow_overflow TINYINT(1) UNSIGNED NOT NULL DEFAULT 0,
-			EVT_timezone_string VARCHAR(45) NULL,
-			EVT_external_URL VARCHAR(200) NULL,
-			EVT_donations TINYINT(1) NULL,
+			EVTM_ID int(10) NOT NULL AUTO_INCREMENT,
+			EVT_ID bigint(20) unsigned NOT NULL,
+			EVT_display_desc tinyint(1) unsigned NOT NULL DEFAULT 1,
+			EVT_display_ticket_selector tinyint(1) unsigned NOT NULL DEFAULT 1,
+			EVT_visible_on datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+			EVT_DEFAULT_registration_status varchar(3),
```

Comment:
This doesn't look right here.  Should be `EVT_default_registration-status` right?

Reply:
bah, yes